### PR TITLE
Complete database Schema and seeding script

### PR DESCRIPTION
**Pull request by tegsheeb**
This pull request completes the FEC deliverable. 
1) Set up node driver for MySQL 
2) Complete database Schema. To make a simpler database, there are 'rooms' and 'images' tables with a one-to-many relationship. 
3) The seeding script completes outputs of 100 items. For simplicity's sake, it uses a random image generator (LoremFlicker) given the topic 'room'. Implementing this functionality with S3 will be a 'tech debt/refactoring' ticket. 
4) The code was run and tested. Verified the output in the MySQL server.

### CODE REVIEW

File: database/Schema.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  number_of_reviews int(5),
+  PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS images;
+
+CREATE TABLE images (
+  id INT NOT NULL auto_increment,
+  image_url varchar(250) NOT NULL,
+  room_id INT,
```
Comment (anaveragejosef) [inline database/Schema.sql:22]: @tegsheeb Is this a foreign key on line 22? If so, do you want to declare it?

Not 100% sure if it is necessary but figured I would ask.

**Review by anaveragejosef — CHANGES_REQUESTED** [review]
Hi @tegsheeb, I just finished going through all the files and it looks good to me.

I see that you added the config file to your gitignore so that you can protect that data and the seed generation and loading all looks good. I also saw that you added the seed script and that works as well.

The only question I have is about the Foreign Key. If you are good with the schema as is, that works for me and I will finish the approval.

File: database/Schema.sql (hunk 1 above)
Comment (tegsheeb) [reply database/Schema.sql:22]: @anaveragejosef Nice catch! yes, it is a foreign key. I need to add 
   FOREIGN KEY (room_id) REFERENCES rooms(id) . 
and just added now.

Comment (anaveragejosef) [reply database/Schema.sql:22]: @tegsheeb Awesome! Looks good to me now. I will resolve this and mark it ready for merge.

**Review by anaveragejosef — APPROVED** [review]
After seeing that you added the foreign key, this is good to go! Approving now.

**Pull request merged**